src/Validation/Validation.php

```diff
@@ -1021,6 +1023,74 @@ public static function uploadedFile($file, array $options = [])
     }
 
     /**
+     * Validates a geographic coordinate.
+     *
+     * Supported formats are right now:
+     *
+     * - <latitude>, <longitude> Example: -25.274398, 133.775136
+     *
+     * ### Options
+     *
+     * - `type` - A string of the coordinate format, right now only `latLong`.
+     * - `latLong` - By default `both`, can be `long` and `lat` as well to validate
+     *   only a part of the coordinate.
+     *
+     * @param string $value Geographic location as string
+     * @param array $options Options for the validation logic.
+     * @return bool
+     */
+    public static function geoCoordinate($value, array $options = [])
```

Comment:
@Marlinc it doesn't make sense to concatenate longitude() and latitude() inside geoCoordinate() because you would have to remove characters from the regex that is generated. I don't see why it is such a huge problem to call geoCoordinate() from within longitude() and latitude(). You don't have to pass any option there. If you guys want it different *you*'re welcome to change it.